src/alg_traits.jl

```diff
@@ -243,3 +243,5 @@ as the maximum order of the algorithm.
 function alg_order(alg::AbstractODEAlgorithm)
     error("Order is not defined for this algorithm")
 end
+
+function alg_interpretation end
```

Comment:
needs a docstring

Reply:
There wasn't any docstring for it in `StochasticDiffEq`. What should we put?